src/rent/FlatRent.java

```diff
@@ -0,0 +1,69 @@
+package rent;
+
+public class FlatRent {
+    private int id;
+    private int area;
+    private double costOfRent;
+    private Owner owner = new Owner(1,"+375 29 588 63 25", "Pavel") ;
+    private Client client = new Client(1,25,"Vladislav",true);
+
```

Comment:
Default owner and client for apartment - it is a really strange idea, they can just don't exist. We need to check it in methods